### PR TITLE
Channel docs

### PR DESCRIPTION
**Pull request by matzew**
Adding list of Eventing channels.

with a generator  /cc @grantr @Harwayne

### CODE REVIEW

File: eventing/channels/channels.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    status: Proof of Concept
+    support: None
+    description: >
+      In-memory channels are a best effort Channel. They should NOT be used in Production. They are useful for development.
+  - name: Apache Kafka
+    url: https://github.com/knative/eventing/tree/master/contrib/kafka/config
+    status: Proof of Concept
+    support: None
+    description: >
+      Channels are backed by [Apache Kafka](http://kafka.apache.org/) topics
```
Comment (samodell) [inline eventing/channels/channels.yaml]:
```suggestion
      Channels are backed by [Apache Kafka](http://kafka.apache.org/) topics.
```

Comment (samodell) [reply eventing/channels/channels.yaml]: Every other description ends with a period, so let's add one here.

**Comment by matzew** [comment]
@samodell thanks, Sam!

**Comment by matzew** [comment]
@samodell due to the madness of the CLA bot, I used your commit and squashed it into mine - is that OK for ya ?

**Pull request merged**